lib/src/charts/painters/cartesian/cartesian_chart_painter.dart

```diff
@@ -169,7 +157,7 @@ class CartesianChartPainter {
       // We will plot texts and point along both X & Y axis
       if (showXLabels && i <= _xUnitsCount) {
         ChartTextPainter.fromChartTextStyle(
-          text: i.toString(),
+          text: i.toPrecision(2).toString(),
```

Comment:
What do we need `toPrecision` for?
And if we do, this might need to be configurable from the widget.

Reply:
The Y Axis label values lerp from previous to next value when the grid shifts. While lerping the divisible value ends up upto 8 digits so to make it look little cleaner, we are rounding upto two digits.